dask/array/tests/test_fft.py

```diff
@@ -27,9 +31,21 @@ def key(k):
     "ihfft",
 ]
 
+all_nd_funcnames = [
+    "fft2",
+    "ifft2",
+    "fftn",
+    "ifftn",
+    "rfft2",
+    "irfft2",
+    "rfftn",
+    "irfftn",
+]
+
 nparr = np.arange(100).reshape(10, 10)
 darr = da.from_array(nparr, chunks=(1, 10))
 darr2 = da.from_array(nparr, chunks=(10, 1))
+darr3 = da.from_array(nparr, chunks=(10, 10))
```

Comment:
There appears to be a shape mismatch with the `irfft[2n]`/`rfft[2n]` functions that I would like to understand.